src/Evolution/Systems/GeneralizedHarmonic/Constraints.cpp

```diff
@@ -368,13 +368,13 @@ void f_constraint_add_term_5_of_25(
     const gsl::not_null<tnsr::a<DataType, SpatialDim, Frame>*> constraint,
     const tnsr::a<DataType, SpatialDim, Frame>& spacetime_normal_one_form,
     const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
-    const tnsr::ia<DataType, SpatialDim, Frame>& d_gauge_function) noexcept {
+    const tnsr::ab<DataType, SpatialDim, Frame>& d_gauge_function) noexcept {
```

Comment:
I don't really like the name `d_gauge_function` because it is not obvious whether it is a 3d or 4d derivative.  In this line it is obvious because of the type, but maybe a better name would make it clearer elsewhere.

Reply:
Agreed, as noted [above](https://github.com/sxs-collaboration/spectre/pull/2528#discussion_r504874946) I've renamed this to `spacetime_d_gauge_function`.